src/main/java/io/buildpacks/example/sample/GreetingController.java

```diff
@@ -7,8 +7,14 @@
 @RestController
 public class GreetingController {
 
```

Comment:
1. Error Handling: The `throwError` method deliberately throws a `RuntimeException`. While this might be suitable for testing error scenarios, in a production application it is recommended to have proper exception handling. Consider using `@ExceptionHandler` to handle exceptions and return a meaningful error response to the client.